generate-fbc.sh

```
@@ -1,5 +1,5 @@
 #!/bin/sh
 
-for OCP_VERSION in $(ls -d v*.*); do
+for OCP_VERSION in $(ls -d v[0-9]*); do
```

Shellcheck (0.11.0)

[warning] Iterating over ls output is fragile. Use globs.

(SC2045)